profiler/src/main/java/com/splunk/opentelemetry/profiler/JfrActivator.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright Splunk Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.splunk.opentelemetry.profiler;
+
+import static com.splunk.opentelemetry.profiler.Configuration.CONFIG_KEY_ENABLE_PROFILER;
+
+import com.google.auto.service.AutoService;
+import io.opentelemetry.instrumentation.api.config.Config;
+import io.opentelemetry.javaagent.spi.ComponentInstaller;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+@AutoService(ComponentInstaller.class)
+public class JfrActivator implements ComponentInstaller {
+
+  private static final Logger logger = LoggerFactory.getLogger(JfrActivator.class.getName());
+
+  @Override
+  public void afterByteBuddyAgent(Config config) {
+    if ("false".equals(config.getProperty(CONFIG_KEY_ENABLE_PROFILER))) {
```

Comment:
Why not use the typed method?

```suggestion
    if (config.getBooleanProperty(CONFIG_KEY_ENABLE_PROFILER, true)) {
```

Also if you just use `false` as the default here you won't need a custom `PropertySource` implementation

Reply:
Nice.  I think I had just forgotten that there were typed methods, thanks for the suggestion!

There will be other properties that justify the property source.  :)